README.md

```diff
@@ -44,47 +43,49 @@ directory.
 
 |    Command    | Description |
 | :-----------: | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-|  **`go-quai`**   | Our main Quai CLI client. It is the entry point into the Quai network (main-, test- or private net), capable of running as a full node (default), archive node (retaining all historical state) or a light node (retrieving data live). It can be used by other processes as a gateway into the Quai network via JSON RPC endpoints exposed on top of HTTP, WebSocket and/or IPC transports. `go-quai --help` for command line options.|
+|  **`go-quai`**   | Our main Quai CLI client. It is the entry point into the Quai network (main-, test- or private net), capable of running as a global node (all chains), base node (subset of chains), archive node (retaining all historical state), or a light node (retrieving data live). It can be used by other processes as a gateway into the Quai network via JSON RPC endpoints exposed on top of HTTP, WebSocket and/or IPC transports. `go-quai --help` for command line options.|
 |  **`test`** | Runs a battery of tests on the repository to ensure it builds and functions correctly.|
 
 ## Running `go-quai`
 
-### Full node on the main Quai network
+### Configuration
+
+Configuration is handled in `network.env.dist` file. You will need to copy or rename the file to `network.env`. The make commands will automatically pull from this file for configuration changes.
```

Comment:
Configuration is handled in `network.env`, not `network.env.dis`. `network.env.dist` is an example env file